src/content/en/fundamentals/accessibility/a11y-for-teams.md

```diff
@@ -0,0 +1,341 @@
+
+![A table with primary use cases as rows and checklist items as columns.](imgs/checklist.jpg)
+
+
+### Evaluating with user studies
+
+Nothing beats sitting down with actual users and observing them as they try to
+use your app. If you're retrofitting accessibility into an existing experience,
+this process can help you quickly identify areas that need improvement. And if
+you're starting a new project, early user studies can help you
+avoid spending too much time developing a feature that is difficult to use.
+
+Aim to incorporate feedback from as diverse of a user population as possible.
+Consider users who primarily navigate with the keyboard, or rely on assistive
+technology like screen readers or screen magnifiers.
+
+## UX Designer
+
+Because people tend to design using their own biases, if you don't have a
+disability and don't have colleagues with disabilities, you might be
+unintentionally designing for only some of your users. As you work, ask yourself
+"what are all the types of users who might rely on this design?" Below are a few
+items a UX designer can aim to address to incorporate more inclusivity into
+their process.
+
+- Content has sufficient color contrast.
+- The tab order is identified.
+- Controls have accessible labels.
+- There are multiple ways to interact with the UI.
+
+### Content has good color contrast
+
+The primary goal of most sites is to convey some information to the user, either
+through written text or images. However, if this content is low contrast, it may
+be difficult for some users (particularly those with a vision impairment) to 
+read. This may end up negatively affecting their user experience. To address
+this concern, aim for all text and images to have sufficient color contrast.
+
+Contrast is measured by comparing the luminance of a foreground and background
+color. For smaller text (anything below 18pt or 14pt bold) a minimum ratio of
+4.5:1 is recommended. For larger text, this ratio can be adjusted to 3:1.
+
+In the image below, the text on the left hand side meets these contrast
+minimums, whereas the text on right hand side is low contrast.
+
+![Side-by-side text samples. One is sufficient contrast, one is low contrast.](imgs/contrast.jpg)
+
+There are a number of tools for measuring color contrast, such as Google’s [Material
+Color Tool](https://material.io/color), [Lea Verou's Contrast Ratio
+app](https://leaverou.github.io/contrast-ratio), and Deque’s
+[aXe](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US).
+
+### Tab order is identified
+
+Tab order is defined as the order in which elements receive focus as the user
+presses the tab key. For users who navigate primarily with a keyboard, the
+tab key is their primary means of reaching everything on screen. Think of it
+like their mouse cursor.
+
+Ideally the tab order should flow from the top of the page to the bottom, with
+more important items appearing higher up in the order. This makes it more
```

Comment:
I changed this to:

"Ideally the tab order should follow the reading order and flow from the top of
the page to the bottom, with more important items appearing higher up in the
order."